src/main/features/dashboard/views/defendant.njk

```diff
@@ -16,7 +16,7 @@
 
       {{ responseStatusForDefendant(claim, DashboardUrlHelper.getStatusUrl(claim), isAfter4pm) }}
 
-      {% if featureToggles.offer and not claim.settlementReachedAt  %}
+      {% if featureToggles.offer and not ClaimStatus == OFFER_SUBMITTED and not ClaimStatus == OFFER_SETTLEMENT_REACHED  %}
```

Comment:
How does this work?

Reply:
If you look into the claim.ts (src/main/app/claims/models/claim.ts) file, i already do that check. This change was for more consistency so we aren't using claim.settlementReachedAt() anymore and instead use OFFER_SETTLEMENT_REACHED

Reply:
but isn't `ClaimStatus` just a global variable? this check doesn't seem to do anything

Reply:
It was my understanding that it checks if the toggle offer is on, it makes sure OFFER_SUBMITTED *for that offer* is not true and OFFER_SETTLEMENT_REACHED *for that offer* is not true and then doesn't give them the option to create an offer.

If you have an offer that has been accepted and remove that code, the defendant can still make an offer

Reply:
how does this code work though? what is the `ClaimStatus` variable?

Reply:
If you look into the claim.ts (src/main/app/claims/models/claim.ts) i validate the conditions that the claim falls under.
For example 
```
 } else if (this.eligibleForCCJ) {
      return ClaimStatus.ELIGIBLE_FOR_CCJ
```

```
get eligibleForCCJ (): boolean {
    if (!toBoolean(config.get<boolean>('featureToggles.countyCourtJudgment'))) {
      return false
    }
```